.env.example

```
@@ -132,12 +132,17 @@ DOCMIND_ANALYSIS__MODE=auto
-DOCMIND_GRAPHRAG__ENABLED=false
-DOCMIND_GRAPHRAG__RELATIONSHIP_EXTRACTION=false
-DOCMIND_GRAPHRAG__ENTITY_RESOLUTION=fuzzy
-DOCMIND_GRAPHRAG__MAX_HOPS=2
-DOCMIND_GRAPHRAG__MAX_TRIPLETS=1000
-DOCMIND_GRAPHRAG__CHUNK_SIZE=1024
+# Global GraphRAG feature flag (router/UI gate)
+DOCMIND_ENABLE_GRAPHRAG=false
+# GraphRAG nested configuration (preferred)
+DOCMIND_GRAPHRAG_CFG__ENABLED=false
+DOCMIND_GRAPHRAG_CFG__RELATIONSHIP_EXTRACTION=false
+DOCMIND_GRAPHRAG_CFG__ENTITY_RESOLUTION=fuzzy
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCMIND_GRAPHRAG_CFG__ENTITY_RESOLUTION key should go before the DOCMIND_GRAPHRAG_CFG__RELATIONSHIP_EXTRACTION key

(UnorderedKey)